kratos/solving_strategies/convergencecriterias/residual_criteria.h

```diff
@@ -177,6 +175,7 @@ class ResidualCriteria
             const TDataType float_size_residual = static_cast<TDataType>(size_residual);
             const TDataType absolute_norm = (mCurrentResidualNorm/float_size_residual);
 
+            KRATOS_INFO_IF("RESIDUAL CRITERION", this->GetEchoLevel() > 1 && rModelPart.GetCommunicator().MyPID() == 0) << " :: [ Initial residual norm = " << mInitialResidualNorm << "; Current residual norm =  " << mCurrentResidualNorm << "]" << std::endl;
```

Comment:
Note that the MyPID() check is not strictly needed, KRATOS_INFO_IF only prints on rank 0 now for MPI

Reply:
I wanted to write the same but I think it makes more sense to update the loggers all at once
=> removing the rank-check and passing the DataComm